apps/preview/next/pages/showcases/Filters/FiltersPanel.tsx

```diff
@@ -353,40 +368,68 @@ export default function FiltersPanel() {
                   </p>
                 </SfListItem>
               ))}
-            {section.type === 'radio' &&
-              section.details.map(({ label, value, counter }) => (
-                <SfListItem
-                  key={value}
-                  as="label"
-                  size="sm"
-                  disabled={counter === 0}
-                  className={classNames('px-1.5 bg-transparent hover:bg-transparent', {
-                    'font-medium': isFilterSelected(value),
-                  })}
-                  slotPrefix={
-                    <SfRadio
-                      disabled={counter === 0}
-                      value={value}
-                      name="radio-price"
-                      checked={isFilterSelected(value)}
-                      onChange={(event) => {
-                        handleRadioSelection(event.target.value);
-                      }}
-                    />
-                  }
-                >
-                  <p>
-                    <span className="mr-2 text-sm">{label}</span>
-                    <SfCounter size="sm">{counter}</SfCounter>
-                  </p>
-                </SfListItem>
-              ))}
+            <fieldset id="radio-price">
+              {section.type === 'radio' &&
```

Comment:
Now the `<fieldset>` is outside so you will get it in every case. It should be inside the condition.
```suggestion
            {section.type === 'radio' && (
              <fieldset id="radio-price">
```